benchmarks/kubectl-mtb/test/benchmarks/configure_ns_quotas/configure_ns_quotas.go

```diff
@@ -0,0 +1,93 @@
+package configurensquotas
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"sigs.k8s.io/multi-tenancy/benchmarks/kubectl-mtb/bundle/box"
+	"sigs.k8s.io/multi-tenancy/benchmarks/kubectl-mtb/pkg/benchmark"
+	"sigs.k8s.io/multi-tenancy/benchmarks/kubectl-mtb/test"
+	"sigs.k8s.io/multi-tenancy/benchmarks/kubectl-mtb/test/utils"
+)
+
+func getResourceNameList(tclient *kubernetes.Clientset) ([]string, error) {
```

Comment:
@Divya063 Using nodes for retrieving resource list is not a good idea here because it contains more resources that we dont have to check like hugepages. We can go with pre-set resource list and giving user feature to add their own resources.

Reply:
@JimBugwadia Would it be okay if we ask the user to pass their own resource list via CLI or a YAML file?